robots/LoCoBot/locobot_control/nodes/pointrobot3factor_ros_server.py

```diff
@@ -52,40 +52,37 @@ class Robot(object):
     Publishes gpmp trajectory to the trajectory action server.
 
     """
+
     def __init__(self):
-        
-        self.ctrl_pub = rospy.Publisher(cmd_vel_topic,
-                                        Twist, queue_size=1)
+
+        self.ctrl_pub = rospy.Publisher(cmd_vel_topic, Twist, queue_size=1)
         self.state = RobotState()
         self.sdf = None
         self.sdf_lock = threading.RLock()
 
         rospy.Subscriber(odom_topic, Odometry, self._odometry_callback)
 
-
         rospy.Subscriber(occ_grid_topic, OccupancyGrid, self._occ_grid_callback)
 
         self.traj_client_ = actionlib.SimpleActionClient(
-            '/turtle/base_controller/trajectory',
-            FollowJointTrajectoryAction,
+            "/turtle/base_controller/trajectory", FollowJointTrajectoryAction,
         )
 
         server_up = self.traj_client_.wait_for_server(timeout=rospy.Duration(10.0))
         if not server_up:
-            rospy.logerr("Timed out waiting for Joint Trajectory"
-                 " Action Server to connect. Start the action server"
-                 " before running example.")
+            rospy.logerr(
+                "Timed out waiting for Joint Trajectory"
+                " Action Server to connect. Start the action server"
+                " before running example."
+            )
             rospy.signal_shutdown("Timed out waiting for Action Server")
             sys.exit(1)
 
-
-
-
-    def _occ_grid_callback(self, msg): 
+    def _occ_grid_callback(self, msg):
         cols = msg.info.width
         rows = msg.info.height
         origin_x = msg.info.origin.position.x
-        origin_y =  msg.info.origin.position.x
+        origin_y = msg.info.origin.position.x
```

Comment:
should this be
`origin_y = msg.info.origin.position.y`

Reply:
Good catch! :)